app/src/main/java/org/mozilla/focus/home/pocket/PocketVideoFragment.kt

```diff
@@ -22,6 +23,11 @@ class PocketVideoFragment : Fragment() {
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
         val layout = inflater.inflate(R.layout.fragment_pocket_video, container, false)
         layout.videoFeed.gridView.adapter = PocketVideoAdapter()
+
+        // SVGs can have artifacts if we set them in XML so we set it in code.
+        layout.pocketWordmarkView.setImageDrawable(context.getDrawable(R.drawable.ic_pocket_and_wordmark).apply {
```

Comment:
As far as I remember, this was only ever a problem for the settings icon, on the FireStick. Is it a problem for this icon too?

Reply:
There is another icon it affected. It seems easier/safer to systematically set all the SVGs in code than to forget to update the code if we update the asset.